README.md

```diff
@@ -636,9 +636,7 @@ def load(filename) do
 end
 ```
 
-#### Pipe Operator. What if we wanted to call some of our functions in succession
-to another. Let's create a function that creates a zoo, randomises it and then
-returns a selected number of animals to go and see:
+#### Pipe Operator. What if we wanted to call some of our functions in succession to another. Let's create a function that creates a zoo, randomises it and then returns a selected number of animals to go and see:
```

Comment:
@jackcarlisle  I _agree_ with having a sub-section for `Pipe Operator` but not a _fan_ of the super-long `<h4>` heading ... could we re-word this?